Resources/Prototypes/Maps/Pools/default.yml

```
@@ -1,20 +1,4 @@
-  - Fland
-  #- Gate
-  - Loop
-  - Marathon
-  - Meta
-  - Oasis
-  - Omega
-  - Packed
-  - Plasma
-  - Reach
-  #- Train
+  maps: # Fire edit - все стандартные карты удалены в качестве оптимизации RAM
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)